app/utils/bunyan.js

```diff
@@ -36,14 +36,15 @@ const getBunyanConfig = (route) => {
       streams: [{
         level: process.env.LOG_LEVEL || 'info',
         stream: process.stdout
-      }]      
+      }]
     };
   }
   let result = {
     name: route,
     parseUA: false,
     excludes: ['referer', 'url', 'body', 'short-body'],
     levelFn: responseCodeMapper,
+    obfuscate: ['razee-org-key'],
```

Comment:
should this be `obfuscate: ['req.headers.razee-org-key'],` instead?

Reply:
probably/ I'll push an update for this as well as to make the config consistent